block-producer/src/block_builder/prover/mod.rs

```diff
@@ -224,11 +259,12 @@ impl BlockProver {
 // =================================================================================================
 
 /// Provides inputs to the `BlockKernel` so that it can generate the new header
-#[derive(Debug, PartialEq, Eq)]
+#[derive(Debug, PartialEq)]
 pub(super) struct BlockWitness {
```

Comment:
Given that the `BlockWitness` has now become quite big, I wonder if we should move it into its own module (e.g., `src/block_builder/prover/block_witness.rs`.

Reply:
agreed - done